__tests__/app.test.js

```diff
@@ -27,3 +28,17 @@ describe("/api/topics", () => {
       });
   });
 });
+
+describe("/api", () => {
+  test("GET:200 Retrieves a list of endpoints", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        expect(Object.keys(response.body).length).toBe(
```

Comment:
we can just expect response.body to equal endpoints here, otherwise were just checking length but dont know the keys